pgzero/actor.py

```diff
@@ -216,6 +224,51 @@ def image(self):
     def image(self, image):
         self._image_name = image
         self._orig_surf = self._surf = loaders.images.load(image)
+        self._adjust_scale_and_angle(self._scale_x, self._scale_y)
+        self._update_pos()
+
+    @property
+    def scale(self):
+        return self._scale_x, self._scale_y
+
+    @scale.setter
+    def scale(self, scale):
```

Comment:
I'd quite like scale to accept a single int/float, as a shortcut for setting both values.

Actually I would expect that is the more common case.

Reply:
done